lib/ansible/modules/net_tools/dnsmadeeasy.py

```diff
@@ -607,6 +609,10 @@ def main():
         new_record["port"] = new_record["value"].split(" ")[2]
         new_record["value"] = new_record["value"].split(" ")[3]
 
+    # Special handling for TXT records (escape extra quotes)
+    if new_record["type"] == "TXT":
+        new_record["value"] = '\"{}\"'.format(new_record["value"])
```

Comment:
How will this relate to the scenario when the quotes have already been explicitly inputed?

```
record_value: '"foo bar baz"'
```

Something which is likely needed when you create a TXT record value consisting of two strings.


```
record_value: '"foo bar baz" "bar foo"'
```

Reply:
Valid point. Need to create a unit test for this module to catch all possible scenarios. According to the RFC, additional quotes within the value must be escaped via accent grave as well as equal signs. The current patch makes the standard use case usable. But needs additional love. True.

Reply:
@EagleIJoe: Feel free to ping me if you have any DNS format/specification/behavior questions. While I really don't know anything about the DNS Made Easy API I do know quite a bit about DNS in general.

I'm __andol__ on IRC.